packages/react-components/src/status/status.css

```diff
@@ -0,0 +1,68 @@
+.status-widget {
+  display: flex;
+  height: 100%;
+  box-sizing: border-box;
+  flex-direction: column;
+  padding: var(--margin-medium);
```

Comment:
Do we consider using [@cloudscape/design-token](https://cloudscape.design/foundation/visual-foundation/design-tokens/)? Or a bigger question: how much do we want iot-app-kit to look like cloudscape components?

Reply:
yeah i think that's a good callout - right now the styling is ported from synchro charts but i think thats a good goal to move towards